docs/FOR VALIDATORS/Installation & Quick Start/imversed.md

```diff
@@ -0,0 +1,166 @@
+The Cosmos SDK automatically generates two configuration files inside `~/.imversed/config`:
+
+* `config.toml`: used to configure the Tendermint, learn more on [Tendermint's documentation](https://docs.tendermint.com/master/nodes/configuration.html),
+* `app.toml`: generated by the Cosmos SDK, and used to configure your app, such as state pruning strategies, telemetry, gRPC and REST servers configuration, state sync, JSON-RPC, etc.
+
+Both files are heavily commented, please refer to them directly to tweak your node.
+
+One example config to tweak is the `minimum-gas-prices` field inside `app.toml`, which defines the minimum amount the validator node is willing to accept for processing a transaction. It is an anti spam mechanism and it will reject incoming transactions with less than the minimum gas prices.
+
+If it's empty, make sure to edit the field with some value, for example 10token, or else the node will halt on startup.
+
+```text
+ # The minimum gas prices a validator is willing to accept for processing a
+ # transaction. A transaction's fees must meet the minimum of any denomination
+ # specified in this config (e.g. 0.25token1;0.0001token2).
+ minimum-gas-prices = "0aimv"
+```
+
+### Pruning of State
+
+There are four strategies for pruning state. These strategies apply only to state and do not apply to block storage. To set pruning, adjust the pruning parameter in the `~/.imversed/config/app.toml` file. The following pruning state settings are available:
+* `everything`: Prune all saved states other than the current state.
+* `nothing`: Save all states and delete nothing.
+* `default`: Save the last 100 states and the state of every 10,000th block.
+* `custom`: Specify pruning settings with the `pruning-keep-recent`, `pruning-keep-every`, and `pruning-interval` parameters.
+
+By default, every node is in `default` mode which is the recommended setting for most environments. If you would like to change your nodes pruning strategy then you must do so when the node is initialized. Passing a flag when starting `imversed` will always override settings in the `app.toml` file, if you would like to change your node to the `everything` mode then you can pass the `---pruning everything` flag when you call `imversed` start.
+
+> **IMPORTANT:** When you are pruning state you will not be able to query the heights that are not in your store.
+
+### Client configuration
+We can view the default client config setting by using `imversed config` command:
+
+```text
+imversed config
+{
+ "chain-id": "",
+ "keyring-backend": "os",
+ "output": "text",
+ "node": "tcp://localhost:26657",
+ "broadcast-mode": "sync"
+}
+```
+We can make changes to the default settings upon our choices, so it allows users to set the configuration beforehand all at once, so it would be ready with the same config afterward.
+
+For example, the chain identifier can be changed to `imversed_9000-4` from a blank name by using:
+
+```text
+imversed config "chain-id" imversed_9000-4

```

Comment:
Change **imversed_9000-4** to **imversed_5555555-1**

Reply:
Updated